packages/react/src/components/Checkbox/Checkbox.mdx

```diff
@@ -0,0 +1,84 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { Checkbox, CheckboxSkeleton } from '../Checkbox';
+
+# Checkbox
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Skeleton state](#skeleton-state)
+- [Component API](#component-api)
+- [Feedback](#feedback)
+
+## Overview
+
+You can build a checkbox using the `Checkbox` in combination with a `<fieldset>` element to group controls and `<legend>` element for the label.
+
+<Preview>
+  <Story id="checkbox--checkbox" />
+</Preview>
+
+## Skeleton state
+
+You can use the `CheckboxSkeleton` component to render a skeleton variant of a checkbox/ This is useful to display while content in your breadcrumb is being fetched from an external resource like an API.
```

Comment:
Referencing `Breadcrumb` here. Doesn't seem applicable to the Checkbox, the skeleton states would probably be used on initial page load